contentcuration/contentcuration/utils/user.py

```diff
@@ -49,3 +50,12 @@ def cache_multiple_users_metadata(users):
         user_id = user["id"]
         key = CACHE_USER_KEY.format(user_id)
         calculate_user_metadata(key, user_id)
+
+
+def calculate_user_storage(user_id):
+    from contentcuration.tasks import calculate_user_storage_task
+
+    key = CACHE_USER_STORAGE_KEY.format(user_id)
+    if key not in cache:
+        cache.set(key, True, timeout=600)  # Invalidate after 10 minutes
+        calculate_user_storage_task.delay(user_id)
```

Comment:
Can add `countdown=5` kwarg here if we want to add a minimum of a 5 second delay to task execution.